core/trino-main/src/main/java/io/trino/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -647,6 +648,24 @@ public PlanAndMappings visitTableExecute(TableExecuteNode node, UnaliasContext c
             return new PlanAndMappings(rewrittenTableExecute, mapping);
         }
 
+        @Override
+        public PlanAndMappings visitSimpleTableExecuteNode(SimpleTableExecuteNode node, UnaliasContext context)
+        {
+            Map<Symbol, Symbol> mapping = new HashMap<>(context.getCorrelationMapping());
+            SymbolMapper mapper = symbolMapper(mapping);
+            // TODO as node.getOutput() is produced by the SimpleTableExecute itself it may be incorrect
```

Comment:
Ask @martint or @kasiafi to review this. We shouldn't land a new feature with a basic TODO like this.